README.md

```diff
@@ -13,21 +12,45 @@ Teams spend significant time building parts of the design system (colors, variab
 [![code style: prettier](https://img.shields.io/badge/code_style-prettier-ff69b4.svg?style=flat-square)](https://github.com/prettier/prettier)
 ![license](https://img.shields.io/github/license/mashape/apistatus.svg)
 
+:star: We appreciate your star, it helps!
+
+## Key features & benefits
+### Unopionated
+These days, there are a million and one great ways to build your next Web site or application. So, we have made being unopinionated the center of every decision made in FAST-DNA. 
+
+This principle of being unopinionated manifests in several important ways including:
+* A flat component architecture that lets you compose what you need without struggling with rigid patterns and complex objects.
+* Separating base components and style that supports multiple frameworks without re-writing or duplicating styles. This separation also lets you build your own styles, with your style technology of choice, without having to re-build or duplicate the base components.
```

Comment:
"style technology", did you intend "style or technology" ?

Reply:
I believe "style technology" refers to JSS, Glamour or other "style technology"

Reply:
Jane is correct, updated to say "styling technology" which I think helps clarify.